textile.go

```diff
@@ -422,6 +554,15 @@ func startNode(cafeOpts CafeOptions, gatewayOpts GatewayOptions) error {
 				if !ok {
 					return
 				}
+				if cafeOpts.SlackHook != "" && cafeOpts.Addr != "" {
```

Comment:
Very easy to add here, limited to only photos, but we could defs supports comments, etc. if we wanted.